content/javascript/concepts/dom-manipulation/terms/createElement/createElement.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.createElement()'
+Description: 'Generates a new element node of the specified type.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'JavaScript'
+  - 'DOM'
+  - 'Methods'
+  - 'Elements'
+CatalogContent:
+  - 'learn-javascript'
+  - 'paths/computer-science'
+---
+
+In JavaScript, the **`.createElement()`** [method](https://www.codecademy.com/resources/docs/javascript/methods) under the `document` [object](https://www.codecademy.com/resources/docs/javascript/objects) is used to generate a new element node of the specified type.
+
+## Syntax
+
+```pseudo
+document.createElement(type)
+```
+
+- `type`: The type of element to create.
+
+## Example
+
+The following example demonstrates the usage of the `.createElement()` method:
+
+```javascript
+document.body.onload = addElement;
```

Comment:
The `document.body.onload` assignment should be removed, as it's unnecessary in most cases. The function should be called explicitly in the script or executed within a `<script>` tag in an HTML document.

```suggestion
```